### PR TITLE
Bug fixes for multiple tickets

### PR DESCRIPTION
**Pull request by georgegomez144**
Encompasses:

- https://jira.lsstcorp.org/browse/EPO-2613
- https://jira.lsstcorp.org/browse/EPO-2610
- https://jira.lsstcorp.org/browse/EPO-2611
- https://jira.lsstcorp.org/browse/EPO-2615
- https://jira.lsstcorp.org/browse/EPO-2618
- https://jira.lsstcorp.org/browse/EPO-2630

### CODE REVIEW

File: src/components/charts/galaxySelector/blinker/BlinkerControls.jsx [hunk 1]
```diff
@@ -20,19 +21,19 @@ const BlinkerControls = ({
...
         icon
-        iconEl={<StartStopTag />}
+        iconEl={<ButtonIcon srText="Start/Stop" Icon={StartStopTag} />}
         onClick={handleStartStop}
       />
       <Button
         className={control}
         icon
-        iconEl={<FastForward />}
+        iconEl={<ButtonIcon srText="Fast Forward" Icon={FastForward} />}
```
Comment (blnkt) [inline src/components/charts/galaxySelector/blinker/BlinkerControls.jsx]: This seems to be working well.  THe other spot that could use this treatment is the mini NavDrawer menu items.  Those all just read out as "Toggle button", as well as the button that opens and closes the full sized NavDrawer.  The latter seems to have a span inside it that should read out in a screen reader correctly, but doesn't for some reason. The case of the former is more critical than the latter as the fullsized NavDrawer is not in fact accessible at all, but that's my bad, an oversight I'll have to deal with at a later date.  But for now, if possible, I think we'd like to see those icon only NavDrawer items get screen reader only text if at all possible within the confines of our react-md world.

File: src/components/charts/shared/navDrawer/nav-drawer.module.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 .nav-drawer-container {
   position: relative;
   overflow: hidden;
+  z-index: 1;
```
Comment (blnkt) [inline src/components/charts/shared/navDrawer/nav-drawer.module.scss]: z-index should come before overflow (see order/properties-order linter warning in terminal)

File: src/components/charts/shared/observationsTables/ObservationsTable.jsx [hunk 3]
```diff
@@ -59,6 +59,7 @@ class ObservationsTable extends React.PureComponent {
       <div className={this.getPositionClassname(position)}>
         {title && <h1 className={styles.tableTitle}>{title}</h1>}
         <Table
+          tabindex="0"
```
[the remark this answers is not in the document]
Comment (georgegomez144) [reply src/components/charts/shared/observationsTables/ObservationsTable.jsx]: All set

File: src/components/charts/shared/navDrawer/nav-drawer.module.scss (hunk 2 above)
Comment (georgegomez144) [reply src/components/charts/shared/navDrawer/nav-drawer.module.scss]: ran `yarn styles`

**Pull request merged**